web/build.gradle

```diff
@@ -40,10 +40,10 @@ dependencies {
     provided group: 'javax.servlet', name: 'javax.servlet-api', version: '3.0.1'
     provided group: 'javax.enterprise', name: 'cdi-api', version: '1.1' // Java EE
     testCompile group: 'junit', name: 'junit', version: '4.12'
-    testCompile group: 'org.mockito', name: 'mockito-all', version: '1.8.0'
-    testCompile group: 'org.eclipse.jetty', name: 'jetty-server', version: '7.0.0.M0'
-    testCompile group: 'org.eclipse.jetty', name: 'jetty-servlet', version: '7.0.0.M0'
-    testCompile group: 'org.eclipse.jetty', name: 'jetty-servlets', version: '7.0.0.M0'
+    testCompile group: 'org.mockito', name: 'mockito-all', version: '1.10.19'
```

Comment:
Added newer version of Mockito which supports annotation and Dependency injection. Makes testing easier.